cranelift-codegen/meta/src/isa/x86/recipes.rs

```diff
@@ -2878,6 +2890,20 @@ pub(crate) fn define<'shared>(
             ),
     );
 
+    recipes.add_template_recipe(
+        EncodingRecipeBuilder::new("icscc_fpr", f_int_compare, 1)
+            .operands_in(vec![fpr, fpr])
+            .operands_out(vec![0])
+            // .inst_predicate(has_eq_condition_code)
```

Comment:
It doesn't seem to matter if I place the instruction predicate here on on the `EncodingBuilder` in `encodings.rs`...

Reply:
What does this comment mean -- is the `EncodingRecipe::inst_predicate()` being ignored somewhere?

Reply:
I guess at the time I didn't understand (and wanted feedback on) why sometimes the predicates are attached where the instruction is encoded and sometimes on the recipe. I now realize that they just need to get to the `EncodingBuilder` in some way and I think the better place is in `encodings.rs`. I will remove this.